templates/etc/nginx/conf.d/auditmatica.conf.j2

```diff
@@ -1,9 +1,46 @@
 map $request $loggable {
-    ~*\.css 0;
-    ~*\.js 0;
-    ~/status/ 0; 
+    # By default, everything is audited
     default 1;
+
+    # List of exceptions:
+    # Polling endpoints
+    ~/status/(.*) 0;
+    ~/transfer/status/(.*) 0;
+    ~/ingest/status/(.*) 0;
+    # DataTable column state management.
+    ~/archival-storage/load_state/ 0;
+    ~/archival-storage/save_state/ 0;
+    ~/backlog/load_state/ 0;
+    ~/backlog/save_state/ 0;
+    # Web assets.
+    ~/media/ 0;
+    ~/static/ 0;
+    ~/jsi18n/ 0;
```

Comment:
These likely need to terminate in `(.*)` as well - `auditmatica` looks for whether a URL begins with these values, but the URL itself is longer and variable depending on the file being served